_guides/getting_started.md

```diff
@@ -10,31 +10,60 @@ body_class: tutorial
 resource_icon: /svg/resources/guide.svg
 ---
 
-# A Step by Step Introduction to Linear Regression
+# Getting Started with Linear Regression and Vowpal Wabbit
 
-## A first data-set
+This guide describes how to run Linear Regression with Vowpal Wabbit (VW). It features an overview of a simple regression problem using a VW workflow tutorial with examples. Find information about interacting with VW, including structuring input, understanding output, and learn important VW diagnostics. 
 
-Now, let's create a data-set. Suppose we want to predict whether a house will require a new roof in the next 10 years. We can create a training-set file, `house_dataset` with the following contents:
+## Getting started with Vowpal Wabbit
+
+To install VW—and for more information on building VW from source or using a package manager—see [Getting started](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Getting-started) guide. 
+
+>**Note:** The CB tutorial uses [Vowpal Wabbit Python package](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python). Additional binary packages are available for select platforms. See [Getting started](https://cheng-tan.github.io/vowpalwabbit.github.io/) module on the homepage for more information.
```

Comment:
@kumpera comments: 
supposed is ambiguous as it might mean that it's available from X but not supported.
Support is a loaded technical term that is usually better replaced.
We want to mention Python, we want to highlight Python.

Rewrite to accomodate, will include in CB guide note as well.